src/fidesops/models/saasconnectionconfig.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import (
+    Column,
+    ForeignKey,
+    String,
+)
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.ext.mutable import MutableDict
+from sqlalchemy.orm import relationship
+
+from fidesops.models.connectionconfig import ConnectionConfig
+
+
+class SaaSConnectionConfig(ConnectionConfig):
```

Comment:
Extended the base ConnectionConfig to have a place to store the saas_configs. This seemed appropriate since the base ConnectionConfig also stores the dataset.yml in a JSONB field.

Reply:
After running into some issues that made extending ConnectionConfig more complex than we would like, @seanpreston and I settled on adding an optional saas_config column to the connectionconfig table.

**Reasoning**
If we extend the connectionconfig model and create saasconnectionconfig for connection configs of type "saas", SQLAlchemy expects there to be "polymorphic_identity" for each connection config type (postgres, mongo, etc.).  The workaround would have been to override the "polymorphic_map" for the base class which added unnecessary complexity.
https://stackoverflow.com/questions/38503863/sqlalchemy-orm-polymorphic-single-table-inheritance-with-fallback-to-parent-cl